Project/DAQ_System/Program/Src/Platform/I2C/STM/i2c_stmf4.hpp

```diff
@@ -13,18 +13,18 @@
 
 namespace platform{
 
-class i2cHalWrapperStmf4 : public II2C{
+class i2c_stmf4 : public II2C{
```

Comment:
Consider renaming to something like `I2cStmF4`. Camel case is highly encouraged for class names.